Dockerfile.arm64

```diff
@@ -19,6 +19,10 @@ RUN pip3 install pillow
 RUN pip3 install scipy
 RUN pip3 install tqdm
 RUN pip3 install PyYAML
+RUN pip3 install easydict
+RUN pip3 install future
+RUN pip3 install numpy
+RUN pip3 install scikit-image
```

Comment:
scikit-image should be removed here as all image functions are performed via PIL

Reply:
Fixed.